.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    webextensions: true
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:mozilla/recommended"
+  ],
+  parserOptions: {
+    ecmaVersion: 2017
+  },
+  plugins: [
+    "mozilla"
+  ],
+  root: true,
+  rules: {
+    "mozilla/use-chromeutils-import": "off",
```

Comment:
This will disable the following errors:

1. Please use `ChromeUtils.import` instead of `Cu.import`
1. Please use `ChromeUtils.defineModuleGetter` instead of `XPCOMUtils.defineLazyModuleGetter`